taca/server_status/cli.py

```diff
@@ -13,13 +13,54 @@ def server_status():
 		raise RuntimeError("Configuration missing required entries: server_status")
 
 
-
 # server status subcommands
 @server_status.command()
-@click.option('--credentials', type=click.Path(exists=True), default=os.path.join(os.path.dirname(__file__), 'gdocs_credentials.json'),
+@click.option('--gdocs', is_flag=True, default=True, help="Update the google docs")
+@click.option('--statusdb', is_flag=True, default=False, help="Update the statusdb")
+@click.option('--credentials', type=click.Path(exists=True), default=os.path.join(os.environ.get('HOME'), '.taca', 'gdocs_credentials.json'),
 				 help='Path to google credentials file')
-def disk_space(credentials):
-	""" Checks the available space on all the disks
+def nases(credentials, gdocs, statusdb):
+	""" Checks the available space on all the nases
+	"""
+	disk_space = status.get_nases_disk_space()
+	if gdocs:
+		status.update_google_docs(disk_space, credentials)
+	if statusdb:
+		status.update_status_db(disk_space)
+
+#  must be run on uppmax, as no passwordless ssh to uppmax servers
+@server_status.command()
+@click.option('--disk-quota', is_flag=True, default=True, help="Check the available space on the disks")
+@click.option('--cpu-hours', is_flag=True, default=True, help="Check the usage of CPU hours")
+def uppmax(disk_quota, cpu_hours):
+	"""
+	Checks the quotas and cpu hours on the uppmax servers
 	"""
-	disk_space = status.get_disk_space()
-	status.update_google_docs(disk_space, credentials)
+	disk_quota_data = {}
+	cpu_hours_data = {}
+
+	if disk_quota:
+		disk_quota_data = status.get_uppmax_quotas()
+	if cpu_hours:
+		cpu_hours_data = status.get_uppmax_cpu_hours()
+
+	merged_results = {}
```

Comment:
ok, but won't it work without making a copy? Can I just do `disk_quota_data.update(cpu_hours_data)` ?